modules/backend/widgets/form/partials/_form_tabs.htm

```diff
@@ -12,10 +12,13 @@
     }
 ?>
 <div class="<?= $navCss ?>">
-    <ul class="nav nav-tabs">
+    <!-- Code change to add vertical tabs
+    <ul class="nav nav-tabs" role="tablist" aria-orientation="<?= $navCss=='vertical-tabs'?'vertical':'horizontal'?>" aria-label="Tabs">
+    -->
```

Comment:
On further thought we're unlikely to add vertical tabs for a while so please remove this comment

Reply:
removed.